pages/my/characters.js

```diff
@@ -52,7 +56,7 @@ class MyCharacters extends Component {
 
         {!loading && (
           <ul className="characters">
-            {this.props.characters.map(character => (
+            {Object.values(characters || {}).map(character => (
```

Comment:
Ho, shiiiiiiiiiiiiiiiit... I've never used `Object.values()`. It makes so much sense! 🎊🎉

Reply:
The Object class has a lot of nifty things in it. 👍